### PR TITLE
metrics enabled + secret credentials

### PR DESCRIPTION
**Pull request by pampatzoglou**

### CODE REVIEW

File: charts/cla/templates/deployment.yaml [hunk 1]
```diff
@@ -26,29 +26,33 @@ spec:
...
             value: {{ .Values.claAssistant.host | default "example.com" }}
-          - name: GITHUB_CLIENT
-            value: {{ .Values.github.client | default "CLIENT" }}
-          - name: GITHUB_USER
-            value: {{ .Values.github.user | default "USER" }}
           - name: PROTOCOL
-            value: http
-          - name: GITHUB_ADMIN_USERS
-            value: {{ .Values.github.user | default "GITHUB_ADMIN_USERS" }}
+            value: https
```
Comment (fgimenez) [inline charts/cla/templates/deployment.yaml]: You should use `.Values.claAssisttant.protocol` here.

**Comment by pampatzoglou** [comment]
Cant enable serviceMonitor.enabled: true unless I add the prometheus stack in the chart.

File: charts/cla/templates/deployment.yaml [hunk 2]
```diff
@@ -26,29 +26,33 @@ spec:
...
               secretKeyRef:
                 name: '{{ template "cla.fullname" . }}-github-credentials'
-                key: pass
+                key: user
           - name: GITHUB_TOKEN
             valueFrom:
               secretKeyRef:
                 name: '{{ template "cla.fullname" . }}-github-credentials'
                 key: token
+          - name: GITHUB_ADMIN
```
Comment (fgimenez) [inline charts/cla/templates/deployment.yaml]: What is this env var used for? According to https://github.com/cla-assistant/cla-assistant#supported-environment-variables there's an optional `GITHUB_ADMIN_USERS` is that it?

**Review by fgimenez — CHANGES_REQUESTED** [review]

**Comment by fgimenez** [comment]
> Cant enable serviceMonitor.enabled: true unless I add the prometheus stack in the chart.

You only need to do this on prod environment.

File: charts/cla/templates/deployment.yaml (hunk 1 above)
Comment (pampatzoglou) [reply charts/cla/templates/deployment.yaml]: ok

File: charts/cla/templates/deployment.yaml (hunk 2 above)
Comment (pampatzoglou) [reply charts/cla/templates/deployment.yaml]: yep mistype. get set by GITHUB_ADMINS in circle env.

**Comment by pampatzoglou** [comment]
> > Cant enable serviceMonitor.enabled: true unless I add the prometheus stack in the chart.
> 
> You only need to do this on prod environment.

So we will need to have 2 values yaml or edit the deploy and add:  --set metrics.serviceMonitor.enabled=true

https://github.com/w3f/cla-assistant/blob/metrics/scripts/deploy.sh#L12-L13

wdyt?

**Comment by fgimenez** [comment]
Adding it to the deploy sounds good :+1:

**Review by fgimenez — APPROVED** [review]

**Pull request merged**